lib/src/test/java/com/thenewboston/api/confirmationvalidatorapi/datasource/ConfirmationDataSourceTest.kt

```diff
@@ -189,7 +189,7 @@ class ConfirmationDataSourceTest {
             }
 
             @Test
-            fun `should fetch valid confirmation blocks successfully`() = runBlockingTest {
+            fun `should fetch valid confirmations blocks successfully`() = runBlockingTest {
```

Comment:
Why rename it?

Reply:
Sorry Ian it might be when I merged and didn't look into